omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -1,12 +1,12 @@
 OMERO.server installation on OS X with Homebrew
 ===============================================
 
-
 .. topic:: Overview
 
     This walkthrough demonstrates how to install OMERO on a clean Mac
-    OS X system (10.9 or later) using Homebrew.  Note that this
-    demonstrates how to install OMERO.server *from the source code*
+    OS X system (10.9 or later) using Homebrew.  This demonstrates how to install
+    OMERO.server, either from the downloaded OMERO.server.zip
+    or from the source code
     via Homebrew, in addition to all its prerequisites. It is aimed at **developers**
     since typically MacOS X is not suited for serious server deployment.
 
```

Comment:
@jburel recommends removing the reference to the scripts completely - no guarantee these work @will-moore  (see the sentence below this comment ``These instructions....``